pallets/chain-space/src/lib.rs

```diff
@@ -993,7 +998,7 @@ pub mod pallet {
 		pub fn subspace_create(
 			origin: OriginFor<T>,
 			space_code: SpaceCodeOf<T>,
-			count: u64,
+			count: Option<u64>,
```

Comment:
Similarly this would be required for `approve()` call as well? Even though space is approved during creation, but when it is revoked, there might be a requirement of `approve()` call as well. Where handling of `IsPermissioned` will be required.
Will send a PR if this is required.